lib/simulation/openfisca/mapping/menage.js

```diff
@@ -30,6 +27,7 @@ module.exports = {
                 'locatairefoyer': 7,
                 'sans_domicile' : 8
             };
+            var logement = situation.logement;
```

Comment:
General note: don't hesitate to navigate objects, adding intermediate variables does not always increase readability.

(no action requested here)